test/domains/sungeun/rect/at_operator.chpl

```diff
@@ -3,7 +3,7 @@ config const n = 7;
 var D1 = {1..n-1};
 writeln(D1);
 
-var D2 = D1@n;
+var D2 = D1 @ n;
 writeln(D2);
-var D3 = D2@n;
+var D3 = D2 @ n;
```

Comment:
This makes me a little more worried about this change, actually.  I almost said something on the issue, but it seems like it would be weird to have some operators that required a space (like this one would) and some operators that allowed it to be skipped (like `a+b` does).  I'd prefer to leave the code untouched and update the .bad file to reflect the new output

Reply:
I'll adjust the at_operator.chpl to show both with and without spaces and adjust the .bad file.

As far as the broader question of `@` as an operator, I'm not sure what we will do there, but we can view it as a separate issue for the purposes of this PR.